tests/e2e/cucumber/steps/ui/shares.ts

```diff
@@ -145,3 +147,21 @@ When(
     await pageObject.navigate()
   }
 )
+
+Then(
+  /"([^"]*)" should( not | )be able to see the following shares$/,
+  async function (
+    this: World,
+    stepUser: string,
+    condition: string,
+    stepTable: DataTable
+  ): Promise<void> {
+    const shouldExist = condition.trim() !== 'not'
+    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
+    const shareObject = new objects.applicationFiles.Share({ page })
+    for (const { resource, owner } of stepTable.hashes()) {
+      const isAcceptedSharePresent = await shareObject.isAcceptedSharePresent(resource, owner)
+      expect(isAcceptedSharePresent).toBe(shouldExist)
```

Comment:
This may show like `expected true to be false` if fails. With assert, you can provide some good message if the comparison fails. will help while debugging.

Reply:
i also think that but I can't see using assertion in e2e. @saw-jan did you know why we are not using Assertion in e2e from the beginning?

Reply:
Don't know. Maybe trying to use utils from playwright
Check if you can do this:
```js
expect(isAcceptedSharePresent, "Some message here").toBe(shouldExist)
```

Reply:
see also https://github.com/microsoft/playwright/issues/7816